src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -34,6 +34,10 @@ public LogicManager(Model model) {
     @Override
     public CommandResult execute(String commandText) throws CommandException, ParseException {
         logger.info("----------------[USER COMMAND][" + commandText + "]");
+
+
+
+
```

Comment:
It is ok to use single empty line to separate the code. This is not align to the rest of the code.